.travis.yml

```diff
@@ -31,17 +31,17 @@ jobs:
     # linux (cron only)
     #
     # linux-next (cron only)
-    - name: "ARCH=arm32_v5 LD=ld.lld REPO=linux-next"
-      env: ARCH=arm32_v5 LD=ld.lld REPO=linux-next
+    - name: "ARCH=arm32_v5 LD=ld.lld LLVM_IAS=1 REPO=linux-next"
+      env: ARCH=arm32_v5 LD=ld.lld LLVM_IAS=1 REPO=linux-next
       if: type = cron
     - name: "ARCH=arm32_v6 REPO=linux-next"
       env: ARCH=arm32_v6 REPO=linux-next
       if: type = cron
     - name: "ARCH=arm32_v7 LD=ld.lld REPO=linux-next"
       env: ARCH=arm32_v7 LD=ld.lld REPO=linux-next
       if: type = cron
-    - name: "ARCH=arm64 LD=ld.lld REPO=linux-next"
-      env: ARCH=arm64 LD=ld.lld REPO=linux-next
+    - name: "ARCH=arm64 LD=ld.lld LLVM_IAS=1 REPO=linux-next"
+      env: ARCH=arm64 LD=ld.lld LLVM_IAS=1 REPO=linux-next
```

Comment:
I'm not comfortable enabling this for arm64 yet.  Maybe we can not add `-Wa,-gdwarf-2` when `LLVM_IAS=1` in the top level Makefile?